### ISSUE TITLE
User feedback - notes and comments

### ISSUE THREAD

**Issue by jussi-club-soda** [club-soda/club-soda-guide#207]
- [ ] small UX question: the red box around "List a Venue" - this could be a bit confusing for users. We've already had one person wanting to nominate a venue actually fill in the listing form, thinking that was the thing they were meant to do. I know this comes from Harry's designs, but maybe worth a discussion?

**Comment by iteles** [comment]
Agreed today with @jussi-club-soda and Laura that we will simply remove the border from the 'List a Venue' link in the nav bar as this link is not actually the most important one for users and should not be highlighted as this is confusing.
